scipy/optimize/_shgo.py

```diff
@@ -8,14 +8,14 @@
 import warnings
 from scipy import spatial
 from scipy.optimize import OptimizeResult, minimize
-from scipy.optimize._shgo_lib import sobol_seq
 from scipy.optimize._shgo_lib.triangulation import Complex
+from scipy.stats import qmc
 
 
 __all__ = ['shgo']
 
 
-def shgo(func, bounds, args=(), constraints=None, n=100, iters=1, callback=None,
+def shgo(func, bounds, args=(), constraints=None, n=128, iters=1, callback=None,
```

Comment:
This change is a little tricky - it does change the behavior for existing end user code. I expect that on average this increases the runtime, and it should also slightly increase the number of problems that get solved. Is it possible that this takes user code that was getting a valid solution before, and now fails to get a solution? That would be a problem - if so, changing to `n = None` and only changing to `2**m` points for `sobol` would address that.

Reply:
OK yes I can set it to `n=None`